src/charm.py

```diff
@@ -1149,7 +1165,34 @@ def _swift_config(self):
         swift_config = safe_load(swift_config_str)
         if not swift_config:
             return {}
-        for key in swift_config_key:
+        # legacy version of the WordPress charm accepts the ``url`` options
+        # here's an example of the ``url`` option:
+        # http://10.126.72.107:8080/v1/AUTH_fa8326b9fd4f405fb1c5eaafe988f5fd/WordPress/wp-content/uploads/
+        # which is a combination of swift url, container name, and object prefix
+        # the new WordPress charm will only take the swift url
+        # TODO: instead of user input, lookup swift url using swift client automatically
+        if "url" in swift_config:
+            swift_url = swift_config["url"]
```

Comment:
Would it be worth putting the execution of the `if` statements into a separate method?

Reply:
This section will be removed in the future (see the TODO above), I think I will just leave it here for now.